package.json

```diff
@@ -8,7 +8,8 @@
   ],
   "scripts": {
     "start": "parcel",
-    "build": "parcel build"
+    "build": "parcel build",
+    "clean": "rimraf .cache && rimraf coverage && rimraf dist && rimraf node_modules"
```

Comment:
coverage is a directory connected to jest and testing, I think you can remove it

Reply:
fixed